sdk/cosmos/azure-cosmos-spark_3-2_2-12/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure.cosmos.spark</groupId>
+    <artifactId>azure-cosmos-spark_3_2-12</artifactId>
+    <version>4.6.0-beta.1</version> <!-- {x-version-update;com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;current} -->
+    <relativePath>../azure-cosmos-spark_3_2-12</relativePath>
```

Comment:
If we never plan to release this parent, then once we ship the child pom, wouldn't this error out when customer tries to use it ?
Or would it work because in spark we always shade everything?

Reply:
the parent's packaging is "pom" so it isn't even possible to create a jar out of it - instead it is purely a pom to allow inheriting it. So, I don't think that this causes any harm? What scenario do you think would be broken? When referencing the child pom in some customer's project and building (retrieving artifacts from Maven)?

Reply:
Yes, referencing child's pom in a customer's project will not resolve the parent pom. So we will have to ship it. (this is what I was talking about in our spring experiment), where we will release the parent pom, but not with the intention of customers using it. And never publicize it.

Reply:
Fixed this by removing the parent-section from pom (flattening the dependency-reduced-pom)